File: AN_2/SEM_1/MAP/lab5/src/domain/Utilizator.java
package domain;

import java.util.List;
import java.util.Objects;
import java.util.Vector;

public class Utilizator extends Entity<Long>{
    private String firstName;
    private String lastName;

    public Utilizator(String firstName, String lastName) {
        this.firstName = firstName;
        this.lastName = lastName;
    }

    public String getFirstName() {
        return firstName;
    }

    public String getLastName() {
        return lastName;
    }

    @Override
    public String toString() {
        return "Utilizator{" +
                "prenume='" + firstName + '\'' +
                ", nume='" + lastName + '\'' +
                ", prieteni="  +
                '}';
    }

}

File: AN_2/SEM_2/MPP/c#/lab2_1/domain/Utilizator.cs
namespace lab2_1.domain;

public class Utilizator
{
    public int IdUtilizator { get; set; }
    public string Nume { get; set; }
    public string Parola { get; set; }

    public Utilizator(int id, string nume, string parola)
    {
        IdUtilizator = id;
        Nume = nume;
        Parola = parola;
    }

    public override string ToString()
    {
        return $"Utilizator: {Nume}";
    }
}
    

